spec/models/statistics/total_savings_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 describe Statistics::TotalSavings do
   describe '#to_s' do
-    context 'there are completed auctions' do
+    context 'there are complete and successful auctions' do
```

Comment:
that is what I am doing in the spec below with `create(:auction, :completed, :with_bidders)` -- that factory is for an auction that has delivery past due but is no accepted. You think we should include more?

This brings up a good point, though, which is that I think it's time for us to clean up some of our factories. They are well-named in some cases but I frequently find myself visiting the factory file to understand what a factory is. Making an issue for that now